src/App.test.jsx

```diff
@@ -1,4 +1,19 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
 
-test('Should render the header', () => {
 
+describe('Header', () => {
+test('Should render the header', async () => {
+    render(
+          <App />
+    )
+    
+    screen.getByAltText('Alchemy Logo');
+    // console.log('||||||||ALT||||||||', altText);
+
+    screen.debug();
```

Comment:
these `screen.debug()`'s are kind of like `console.log()`'s.  They are useful for troubleshooting/validation but you can remove them after your test is written out and passing.